RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
+
+This document summarizes the steps to release the `comit-js-sdk`.
+The steps below were recorded when releasing version `0.10.0`.
+You of course have to replace `0.10.0` with your version to be released!
```

Comment:
🔧 I would just remove this line and use a stub like `X.Y.Z` or `<version-number>`.